bootstrap/configure.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
out of curiosity is this boiler plate that comes from a know source? If so, it would be great to provide a link to it, in the case it ever needs to be updated.

Reply:
It's just a bash shebang that my editor adds automatically when I create a new shell script. I am happy to remove it if it causes confusion. I believe it should work on any modern system. https://stackoverflow.com/questions/10376206/what-is-the-preferred-bash-shebang